### ISSUE TITLE
Dynamically add information/rows to the popups

### ISSUE THREAD

**Issue by mapsam** [codeforamerica/fellows-map#22]
Need to do some thinking about this one... but it would be nice to let @beccablazak add columns to the spreadsheet and have that update dynamically on the map.

**Comment by lyzidiamond** [comment]
How is the data added to the map currently?

**Comment by lyzidiamond** [comment]
Nevermind, awesome tabletop greatness. I bet @eeeschwartz would have some thoughts here too!

**Comment by mapsam** [comment]
Yup, coming in through Tabletop! :tada: But it gets a little wonky converting each row to a `geojson` feature [here](https://github.com/codeforamerica/fellows-map/blob/gh-pages/static/fellows-map.js#L55-L75). You need to make sure you add the columns from the tabletop data to the `properties` in the geojson. It's not the worst thing in the world, but definitely could be a bit more dynamic.
